pkg/kubectl/cmd/patch.go

```diff
@@ -36,13 +37,16 @@ Please refer to the models in https://htmlpreview.github.io/?https://github.com/
 // Partially update a node using strategic merge patch
 kubectl patch node k8s-node-1 -p '{"spec":{"unschedulable":true}}'
 
+// Partially update a node from node.json using strategic merge patch
```

Comment:
Shell comments should use `#` instead of `//`. Please change it accordingly. 
Also, clarify that the file just provides an alternative way of identifying the resources to operate on:
```
# Partially update a node identified by node name "k8s-node-1" using strategic merge patch
...
# Partially update a node identified by the file "node.json" using strategic merge patch
...
```